node/include/cocaine/detail/service/node/slave/machine.hpp

```diff
@@ -172,6 +175,8 @@ class machine_t : public std::enable_shared_from_this<machine_t> {
     auto
     uptime() const -> std::chrono::seconds;
 
+    auto
+    isolate_unsafe() -> api::isolate_t *;
```

Comment:
Why do you use an unsafe ptr? May be reference or smart ptr will be better?

Reply:
Yep, just a bad habit, )) reference will fit better, will redo into reference.